### ISSUE TITLE
Pango-ERROR: Harfbuzz version too old

### ISSUE THREAD

**Issue by larvanitis** [evolus/pencil#581]
* Operating system: Arch Linux
* Pencil version: 3.1.0.ga

I am trying to update the AUR package but after building and installing the latest bin version, I get this error when running pencil:

```
(pencil:795923): Pango-ERROR **: 13:15:50.508: Harfbuzz version too old (1.2.7)

[1]    795923 trace trap (core dumped)  pencil
```

According to Pango, this is an Electron issue.
https://gitlab.gnome.org/GNOME/pango/issues/428

According to Electron, this is a downstream software issue. https://github.com/electron/electron/issues/20348
>The only solution is to ask the developers to update to the latest version of electron, that's all.

**Comment by mjourdan** [comment]
Same here, can't run pencil since my last system upgrade:
```
$ rpm -q Pencil
Pencil-3.0.4-49.x86_64
$ grep PRETTY_NAME /etc/os-release 
PRETTY_NAME="Fedora 31 (Workstation Edition)"
```

**Comment by dgthanhan** [comment]
Please check if the problem persists when upgrading to 3.1.0.

**Comment by mjourdan** [comment]
Indeed, seems I actually had several versions of pencil installed at the same time:

```
 $ rpm -q Pencil
Pencil-3.0.4-49.x86_64
$ rpm -q pencil
pencil-3.1.0.ga-1.x86_64
$ which pencil
/usr/local/bin/pencil
$ LANG=C rpm -qf /usr/local/bin/pencil
file /usr/local/bin/pencil is not owned by any package
```

Still, calling the latest version does solve the issue:

```
$ /opt/pencil-3.1.0.ga/pencil
```

Just had to uninstall the older package to get the symlink straight:

```
$ ls -l /usr/bin/pencil 
lrwxrwxrwx. 1 root root 27  6 nov.  01:46 /usr/bin/pencil -> /opt/pencil-3.1.0.ga/pencil
```

Thanks, @dgthanhan !

**Comment by rafaelreuber** [comment]
> Please check if the problem persists when upgrading to 3.1.0.
Uinstall do old version and install the 3.1.0, fixed for me.

**Comment by bjoernv** [comment]
> > Please check if the problem persists when upgrading to 3.1.0

Upgrading to 3.1.0 fixed the problem for me too.
